Tests/TestingTests/PlanTests.swift

```diff
@@ -26,8 +29,11 @@ struct PlanTests {
       testB,
     ]
 
+    let selection = Test.ID.Selection(testIDs: [innerTestType.id])
     var configuration = Configuration()
-    configuration.selectedTestIDs = [innerTestType.id]
+    configuration.testFilter = { test in
+        selection.contains(test)
+    }
```

Comment:
Seems like these could all use your new `setTestFilter(toMatch:)` function and save some boilerplate

Reply:
Good idea! Done!